crypto/keyring/legacy_info.go

```diff
@@ -12,8 +12,8 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// Info is the publicly exposed information about a keypair
-type legacyInfo interface {
+// LegacyInfo is the publicly exposed information about a keypair
+type LegacyInfo interface {
```

Comment:
I made LegacyInfo public to resolve linter issue `exported func NewLegacyMultiInfo returns unexported type github.com/cosmos/cosmos-sdk/crypto/keyring.legacyInfo, which can be annoying to use`

Reply:
Do we need to export `NewLegacyMultiInfo`? Maybe we can remove it?

Reply:
we should export `NewLegacyMultiInfo` cause it is used in `client/keys` package.